roles/openshift_named_certificates/tasks/main.yml

```diff
@@ -40,10 +53,42 @@
     mode: 0600
   with_items: "{{ named_certificates }}"
 
+- set_fact:
+    named_certificates_ca_file_defined: "{{ named_certificates | lib_utils_oo_collect('cafile')}}"
+  
 - name: Land named CA certificates
   copy:
     src: "{{ item }}"
     dest: "{{ named_certs_dir }}/{{ item | basename }}"
     remote_src: "{{ openshift_named_certificates_from_remote | default(False) }}"
     mode: 0600
   with_items: "{{ named_certificates | lib_utils_oo_collect('cafile') }}"
+
+- name: Verify if the ca-bundle contains the CA to trust the named certificate
+  command: >
+    openssl verify -CAfile /etc/origin/master/ca-bundle.crt "{{ named_certs_dir }}{{ item.certfile | basename }}"
+  with_items: "{{ named_certificates }}"
+  ignore_errors: True
+  register: ca_bundle_result
+  when: named_certificates_ca_file_defined | length > 0
```

Comment:
The task just verifies trust the of the named certificates and registers the output. If the ca-bundle.crt doesn't trust the named certs, the following task appends the cafile to the ca-bundle.

I understand that the openshift-master/redeploy-certificates will overwrite that, however, the result at the end is the same.